docs/enterprise/reference/manage.md

```diff
@@ -0,0 +1,297 @@
+---
+title: Manage
+lang: en-US
+sidebarDepth: 2
+meta:
+    - name: keywords
+      content: configuration options settings Pomerium enterprise console
+---
+
+# Manage
+
+## Routes
+
+A Route provides access to a service through Pomerium.
```

Comment:
I think it might be helpful to frame a route as a concept as containing the set of configuration knobs you'd adjust in any other reverse proxy like nginx, haproxy, envoy, or traefik.

Reply:
I think the updated version does a better job of this.

Reply:
Unresolved, because I think I was thinking of Routes in the concepts page. I'll suggest a rephrasing.